doc/man/scons.xml

```diff
@@ -102,35 +102,49 @@ orchestrates the construction of software
 (and other tangible products such as documentation files)
 by determining which
 component pieces must be built or rebuilt and invoking the necessary
-commands to build them.</para>
+commands to build them.
+&SCons; offers many features to improve developer productivity
+such as parallel builds, caching of build artefacts, and
```

Comment:
spelling? artifacts

Reply:
Brtish vs American, but more pointedly, artefact is being used more frequently for the past product of a technical process, as opposed to a piece of physical history.  However, they're virtually interchangeable and I have no problem changing to "artifact"

Reply:
Please change. I've never seen the artefact used in docs before.. I'm sure someone will file a GH issue about spelling without following instructions  and you know how much I LOVE that... ;)

Reply:
done.